platform/_partials/install/multiple-sso.mdx

```diff
@@ -1,12 +1,14 @@
-In vCluster Platform, you can also configure _multiple_ SSO providers together. This can be a very handy feature if you
-have multiple groups using different authentication providers based on team or geographical
-region. Configuration of multiple SSO providers _of the same type_ is also possible.
-The latter is configured by using the "connectors" option. Below is an example vCluster Platform config "auth"
-section that contains three SSO providers (one OIDC, and two GitHub providers). When users log
-into vCluster Platform with such a configuration present, they will see three options for authentication as
-well as the default user/password box (which can be disabled if desired).
+In the Platform, you can also configure _multiple_ SSO providers together. This
+can be a very handy feature if you have multiple groups using different
+authentication providers based on team or geographical region. Configuration of
+multiple SSO providers _of the same type_ is also possible. The latter is
+configured by using the "connectors" option. Below is an example platform config
+"auth" section that contains three SSO providers (one OIDC, and two GitHub
+providers). Users logging into the platform with such a configuration present
+see three options for authentication as well as the default username/password
+box (which you can turn off if desired).
 
-```yaml
+```yaml title="platform-config.yaml with multiple SSO providers"
```

Comment:
I wouldn't call this platform-config.yaml as we don't call it that anywhere else in the docs.